### PR TITLE
Pension 73987 disclosure title

### PR DESCRIPTION
**Pull request by ToddWebDev**
## Summary
Update disclosure titles on the Pension Benefits form (21P-527EZ)

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Go to the dependents section of the 'Household information' chapter
3. Verify that the legend is displayed with the correct information
4. Verify that the screen reader includes the dependents name when announcing the legend on the subsequent dependent pages

## Acceptance criteria
[Issue in Github - 74871](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73987)

## Related issue(s)
n/a

## Testing done
- [x] Existing unit tests updated
- [ ] New unit tests added

## Screenshots
| Before | After |
| ------ | ------ |
|localhost_3001_pension_application_527EZ_ (3)|localhost_3001_pension_application_527EZ_review-and-submit (5)

## What areas of the site does it impact?
Pension Benefits Application

### Quality Assurance & Testing
- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

**Review by TaiWilkin — DISMISSED** [review]
LGTM

File: src/platform/forms/sass/_m-schemaform.scss
Comment (micahchiang) [inline src/platform/forms/sass/_m-schemaform.scss:1]: Was this an intentional addition to this PR? This will likely impact all forms on va.gov.

Comment (ToddWebDev) [reply src/platform/forms/sass/_m-schemaform.scss:1]: It was intentional. The updated legend text color will more closely align with the VADS [typography](https://design.va.gov/foundation/typography) and [color](https://design.va.gov/foundation/utilities/text-color) guidelines and form component header and label text colors and improve text color contrast. This is also the text color used in the VADS storybook example for a form input [label header](https://design.va.gov/storybook/?path=/docs/uswds-va-radio--label-header).  I'm happy to isolate the change to the pensions app if preferred but this seemed to be better served as a global change.

Comment (ToddWebDev) [reply src/platform/forms/sass/_m-schemaform.scss:1]: @micahchiang I've reverted this global text color change as we are going to start implementing the titleUI web component pattern instead.

**Comment by ToddWebDev** [comment]
Relevant changes from this PR are now included with https://github.com/department-of-veterans-affairs/vets-website/pull/28294

**Pull request closed without merging**